pkg/network/ovn_kubernetes.go

```diff
@@ -118,6 +118,11 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	data := render.MakeRenderData()
 	data.Data["ReleaseVersion"] = os.Getenv("RELEASE_VERSION")
 	data.Data["OvnImage"] = os.Getenv("OVN_IMAGE")
+	data.Data["OvnNodeImage"] = os.Getenv("OVN_IMAGE")
```

Comment:
If we are using `OvnNodeImage/OvnMasterImage` for self-hosted I don't think we should keep the `OvnImage` as it becomes confusing with three variables.